cmd/rancherd/auth/auth.go

```diff
@@ -29,7 +30,8 @@ var (
 	defaultAdminLabel      = map[string]string{defaultAdminLabelKey: defaultAdminLabelValue}
 )
 
-func ResetAdmin(c *cli.Context) error {
+func ResetAdmin(_ *cli.Context) error {
```

Comment:
why do we drop to not use context from root command?

Reply:
RKE2 reverted back to urfave/cli v1 which doesn't provide a context.Context implementation nor member.